backend/index.js

```diff
@@ -18,6 +23,13 @@ client.on('error', err => {
 app.use(cors());
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
+app.use(cookieParser())
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Methods", "GET, PUT, POST");
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+  next();
+});
```

Comment:
- [ ] For the final version, create a .cofing file / environment variable to specify the domain so that we can enforce a strict domain policy

Reply:
We should do this and create a issue ticket